src/Stats.ts

```diff
@@ -96,11 +96,12 @@ export default class Stats {
 			}
 		});
 
+		// TODO
 		const hooks: Record<string, boolean> = component && {
-			oncreate: !!component.templateProperties.oncreate,
-			ondestroy: !!component.templateProperties.ondestroy,
-			onstate: !!component.templateProperties.onstate,
-			onupdate: !!component.templateProperties.onupdate
+			oncreate: false,
+			onDestroy: false,
+			onstate: false,
+			afterUpdate: false
 		};
 
 		return {
```

Comment:
In my local copy, I'm exposing `component && component.expectedProperties` here for my linter plugin, but I don't know what it should be called. The simple answer is just `expectedProperties` but that doesn't feel quite right.

Reply:
Perhaps `templateReferences`.